src/test/java/teammates/test/cases/datatransfer/FeedbackMsqQuestionDetailsTest.java

```diff
@@ -272,7 +272,8 @@ public void testGetMsqOtherWeight_nullOtherWeight_exceptionThrown() {
         requestParams.put(Const.ParamsNames.FEEDBACK_QUESTION_MSQ_WEIGHT + "-0", new String[] { "2.57" });
         requestParams.put(Const.ParamsNames.FEEDBACK_QUESTION_MSQ_WEIGHT + "-1", new String[] { "1.12" });
         requestParams.put(Const.ParamsNames.FEEDBACK_QUESTION_MSQOTHEROPTIONFLAG, new String[] { "on" });
-        // Removed to send null as otherWeight parameter
+        // The following line is commented out to create the situation for the test,
+        // Where otherWeight is null, as the otherWeight parameter is missing from the requestParams.
```

Comment:
// The following line is commented out, so otherWeight parameter is missing from the requestParams.